lib/adal.js

```diff
@@ -317,6 +336,11 @@ var AuthenticationContext = (function () {
         return this._user;
     };
 
+    /**
+     * If multiple requests for acquireToken are made for the same resource, we send only the first request and register the callbacks for all subsequent requests.
+     * When we process the response from AAD, this method calls all the callbacks with the error/token.
```

Comment:
This should describe what this method does. All it does is add the passed callback to the array of callbacks for the specified resource and put the array on the window object. It does not call the callbacks itself.